src/amo/pages/Addon/index.js

```diff
@@ -544,20 +543,7 @@ export class AddonBase extends React.Component {
                     isAddonEnabled={isAddonEnabled}
                   />
                 )}
-                {showGetFirefoxButton && (
-                  <Button
-                    buttonType="confirm"
-                    href={`https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM(
-                      {
-                        utm_content: addon.guid,
-                      },
-                    )}`}
-                    puffy
-                    className="Button--get-firefox"
-                  >
-                    {i18n.gettext('Only with Firefox—Get Firefox Now')}
-                  </Button>
-                )}
+                {showGetFirefoxButton && <GetFirefoxButton addon={addon} />}
```

Comment:
this is nice:)

quick question: eventually this compatibility check (`showGetFirefoxButton`) will move to the `InstallButtonWrapper` (per our discussion), right?

Reply:
Yes